backend/src/middleware/auth.js

```
@@ -299,34 +312,41 @@ export const validateOwnership = (resourceType) => {
           service = FishService;
           break;
         case 'decoration':
-          const { DecorationService } = await import('../services/decorationService.js');
+          const { DecorationService } = await import(
+            '../services/decorationService.js'
+          );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)